blogs/2025/11/03/universe-recap.md

```diff
@@ -0,0 +1,118 @@
+
+> Pro tip: You can find hundreds of custom instructions, prompt files and agents over on the [awesome-copilot](https://github.com/github/awesome-copilot) repo. If you haven't checked that out yet, you're missing out. It's a treasure trove of inspiration and ready-made prompts.
+
+And If you read the Plan prompt, you'll see it make a tool call to something called "[runSubagent](https://code.visualstudio.com/updates/v1_105#_isolated-subagents)".
+
+### Subagents
+
+[Context Confusion](https://www.dbreunig.com/2025/06/26/how-to-fix-your-context.html) is a real problem with agents. The more you interact, the more context they track - and the more likely they are to get confused. There's a whole new discipline for managing context called "Context Engineering".
+
+With the latest VS Code release, we've added a tool called "[runSubagent](https://code.visualstudio.com/updates/v1_105#_isolated-subagents)" to help you manage context.
+
+[Subagents](https://code.visualstudio.com/updates/v1_105#_isolated-subagents) run independently from the main chat and have their own context. You can call one by adding the `#runSubagent` tool to your prompt. The LLM creates a prompt, hands it off to a subagent, and that agent only gets the context you send. It knows nothing about the rest of your chat, and your chat knows nothing about the subagent's context. Subagents don't pause for feedback and have access to most of the same tools as the main chat.
+
+When a subagent finishes, it returns the final result to the main chat - and only that result joins the main context. Subagents keep your main chat lean while letting you go on sidebars and deep dives. For example, if you're building an API and need to research authentication, spin up a subagent to do that.
+
+```
+Analyze the #file:api with #runSubagent and recommend the best authentication strategy for a web client consuming these endpoints.
+```
+
+You'll know a subagent is running because you can see tool calls and model responses below the subagent action. In the screenshot below, that's "Analyze app structure for auth".
+
+![A subagent process running in VS Code with tool calls underneath the main agent action](subagent-in-progress.png)
+
+We're still exploring ways to help you manage context with agents, and subagents are just the beginning.
+
+## Looking Ahead
+
+Agents are changing how we write code and how we work. You shouldn't have to pick just one. You should be able to move between agents, keep fine-grained control over your context, and create your own Agents to extend our system prompts.
+
+These are just a few highlights from this year's [GitHub Universe](https://github.com/events/universe). Check out [GitHub's blog](https://github.blog/) for all the updates as we work on a unified workflow for a multi-agent experience everywhere you need it.
```

Comment:
is github's blog the main CTA we want to have here? feels like we should be linking to vs code release notes or blog instead